src/validator.rs

```diff
@@ -1,11 +1,11 @@
 use crate::{MpesaError, MpesaResult};
 
 pub trait PhoneValidator {
-    fn validate(&self) -> MpesaResult<()>;
+    fn validate_number(&self) -> MpesaResult<()>;
 }
 
 impl PhoneValidator for &str {
-    fn validate(&self) -> MpesaResult<()> {
+    fn validate_number(&self) -> MpesaResult<()> {
         if self.starts_with("254")
```

Comment:
should account for numbers starting with `07*` and `011*` too

Reply:
I am not sure if we want numbers starting with 07 or 011 mainly because Safaricom has a guideline that number should be written like this 2547XXXXXXXXX.